lib/modules/asset/types/AssetApi.ts

```diff
@@ -61,7 +67,8 @@ export type ApiAssetMetadataReplaceResult = KDocument<AssetContent>;
 export interface ApiAssetUpsertRequest extends AssetsControllerRequest {
   action: "upsert";
 
-  _id: string;
+  // TODO it's unused in action, maybe should remove it
+  _id?: string;
```

Comment:
If this is scritly internal structure and not used, shouldn't we just remove now?

Reply:
It's seems to be not used. In any case, it's not used in the action, so even if it passed in request it's useless.
I have verified on all GitHub repository and I don't see any usage of this type, so it seems currently not used by ours projects.
IMHO, yes we can remove it now, because that don't create breaking change of behavior, but if this type it's used somewhere it will be necessary to update the code to fix the type error thrown.

Reply:
Let's remove it then

Reply:
Ok, it's fixed on type and documentation for asset and devices (same mistake)